src/sentry/api/endpoints/group_details.py

```diff
@@ -206,24 +199,64 @@ def get(self, request, group):
         if last_release:
             last_release = self._get_release_info(request, group, last_release)
 
-        try:
-            environment_id = self._get_environment_id_from_request(
-                request, group.project.organization_id)
-        except Environment.DoesNotExist:
-            get_range = lambda model, keys, start, end, **kwargs: \
-                {k: tsdb.make_series(0, start, end) for k in keys}
-            tags = []
-            user_reports = UserReport.objects.none()
-
-        else:
+        # TODO(jess): This is a little gross right now, but I think we should be able
+        # to switch over to using the snuba group serializer soon, but I think at the
+        # very least, i need to update the snuba tagstore backend
+        # `get_group_seen_values_for_environments` to use GroupEnvironment.first_seen so that
+        # first_seen is correct before we force it on everyone. Also, right now, the way we
+        # handle passing invalid environments is different.
+        use_snuba = request.GET.get('enable_snuba') == '1'
+        environment_ids = None
+        environment_id = None
+        tags = []
+        user_reports = UserReport.objects.none()
+
+        if use_snuba:
+            environment_ids = [
```

Comment:
i was going to let it remain `None` for now, because that would include from all environments. but i should probably just merge this first, because then we can actually support multi env https://github.com/getsentry/sentry/pull/11793